composer.json

```diff
@@ -9,7 +9,10 @@
   "license": "Apache-2.0",
   "require": {
     "php": ">=5.6",
-    "monolog/monolog": "^1.16"
+    "monolog/monolog": ">=1.16",
```

Comment:
```suggestion
    "monolog/monolog": "^1",
```

Reply:
If you approve this, remember to run `composer update` to get a new `composer.lock` file

Reply:
do we need to restrict the usage of monolog v2? I think we can let them use version 1 and 2 as well.

Reply:
@msilvagarcia ?